manifests/init.pp

```diff
@@ -357,20 +364,19 @@
   }
   include pulpcore::plugin::certguard # Required to be present by Katello when syncing a content proxy
 
-  $rhsm_url = "https://${servername}:${rhsm_port}${rhsm_path}"
+  $rhsm_url = "https://${client_facing_servername}:${rhsm_port}${rhsm_path}"
 
   class { 'foreman_proxy::plugin::pulp':
     pulpcore_enabled      => true,
     pulpcore_mirror       => $pulpcore_mirror,
-    pulpcore_api_url      => "https://${servername}",
```

Comment:
We should continue passing in the API URL, just like it was prior to this patch.